README.md

```diff
@@ -23,21 +23,101 @@
-conflicts](https://martinvonz.github.io/jj/latest/conflicts)), with features not
-found in most of them
-([working-copy-as-a-commit](https://martinvonz.github.io/jj/latest/working-copy),
-[undo functionality](https://martinvonz.github.io/jj/latest/operation-log),
-automatic rebase, [safe replication via `rsync`, Dropbox, or distributed file
-system](https://martinvonz.github.io/jj/latest/technical/concurrency)).
+Jujutsu is a powerful [version control system](https://en.wikipedia.org/wiki/Version_control)
+for software projects. You use it to get a copy of your code, track changes
+to the code, and finally publish those changes for others to see and use.
+It is designed from the ground up to be easy to use—whether you're new or
+experienced, working on brand new projects alone, or large scale software
+projects with large histories and teams.
+
+Jujutsu is unlike most other systems, because internally it abstracts the user
+interface and version control algorithms from the *storage system* used to
+serve your content. This allows it to serve as a VCS with many possible physical
+storage backends, that may have their own data or networking models—like
+[Mercurial] or [Breezy], or hybrid systems like Google's cloud-based system,
+"Code in the Cloud".
+
+[Mercurial]: https://www.mercurial-scm.org/
+[Breezy]: https://www.breezy-vcs.org/
+
+Today, we use Git repositories as a storage layer to serve and track content,
+making it **compatible with many of your favorite Git-based tools, right now!**
+All core developers use Jujutsu to develop Jujutsu, right here on GitHub. But it
+should hopefully work with your favorite Git forges, too.
+
+We combine many distinct design choices and concepts from other version control
+systems into a single tool. Some of those sources of inspiration include:
+
+- **Git**: We make an effort to [be fast][perf] like Git, from a snappy
+  UX to efficient algorithms. The default storage backend uses Git repositories
+  for "physical storage", for wide interoperability and ease of onboarding.
+
+- **Mercurial & Sapling**: There are many Mercurial-inspired features, such as
+  the [revset] language to select commits. There is [no explicit index][no-index]
+  or staging area. Branches are "anonymous" like Mercurial, so you don't need
+  to make up a name for each small change. Primitives for rewriting history are
+  powerful and simple. Formatting output is done with a robust template language
+  that can be configured by the user.
+
+- **Pijul & Darcs**: Jujutsu keeps track of conflicts as [first-class objects][conflicts]
+  in its model; they are first-class in the same way commits are, while
+  alternatives like Git simply think of conflicts as textual diffs. While not
+  as rigorous as patch-based systems like Pijul and Darcs (which formalize
+  the underlying theory of three-way merges), the effect is that many forms of
+  conflict resolution can be performed and propagated automatically.
```

Comment:
FYI, I mentioned this PR in Pijul's chat, just as a heads up for them.

Reply:
A suggestion I go there was to replace the "formalize the underlying theory of three-way merges"  by "formalize the theory of patches (as opposed to snapshots)". (They don't do 3-way merges.)